lib/hubspot/meeting.rb

```diff
@@ -31,16 +31,30 @@ def find(id)
         new(response)
       end
 
-      def find_by_contact(contact_id)
+      def find_by_contact(contact_id, opts={})
+        params = {
+          limit: opts[:limit].presence,
+          after: opts[:after].presence
+        }.compact
+
+        default_filters = [{ propertyName: 'associations.contact', 'operator': 'EQ', value: contact_id }]
+        default_sorts = [{ propertyName: "hs_lastmodifieddate", direction: "DESCENDING" }]
+
         response = Hubspot::Connection.post_json(MEETING_SEARCH_PATH, {
             params: {},
             body: {
+              **params,
               properties: BASE_PROPERTIES,
-              filters: [{ propertyName: 'associations.contact', 'operator': 'EQ', value: contact_id }]
+              filters: (opts[:filters].presence || []) + default_filters,
+              sorts: opts[:sorts].presence || default_sorts
             }
           }
         )
-        response['results'].map { |f| new(f) }
+
+        {
+          after: response.dig('paging', 'next', 'after'),
+          meetings: response['results'].map { |f| new(f) }
+        }
```

Comment:
If we change the result structure of this methods, we should put a big warning about breaking change in next release (although I doubt many people rushed to use this new method)